front/src/modules/scenario/components/ScenarioExplorer/ScenarioExplorer.tsx

```diff
@@ -23,7 +23,7 @@ const ScenarioExplorer = ({
 }: ScenarioExplorerProps & {
   displayImgProject?: boolean;
 }) => {
-  const { t } = useTranslation('common/scenarioExplorer');
+  const { t } = useTranslation('stdcm');
```

Comment:
nit : we could do this to avoid `scenarioExplorer.` everywhere
```suggestion
  const { t } = useTranslation('stdcm', { keyPrefix: 'scenarioExplorer' });
```

Reply:
Done